src/services/expenses.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+let expenses = [];
+
+function getAll() {
+  return expenses;
+}
+
+function getExpensById(expensId) {
+  const foundExpens = expenses.find(expens => expens.id === Number(expensId));
+
+  return foundExpens || null;
+}
+
+function create(userId, spentAt, title, amount, category, note) {
+  const id = expenses.length
+    ? Number(Math.max(...expenses.map((expens) => expens.id)) + 1)
```

Comment:
make a separate function for `getMaxId` or `getNextId` & reuse it

Reply:
do it
Create a separate file like a `helpers` for it

Reply:
forgot about it on previous time